### ISSUE TITLE
The paper review assignment

### ISSUE THREAD

**Issue by qiaoyuet** [STAT540-UBC/Discussion#303]
Since the course website says the due date is next Wednesday (Feb. 20) and it wasn't mentioned in class today, I just want to confirm: do we follow what's under Course Work -> Paper Review or will we expect any updates (e.g. the link to the paper)? Thank you.

**Comment by saramostafavi** [comment]
Correct, the paper review is due Feb 20. The link in the webpage is correct. This is the paper that describes the data you are analyzing in the assignment. The deadline is set so to encourage you to read and think about the paper in advance of looking at the data in the assignment.
